src/Controllers/Controller.php

```diff
@@ -10,7 +10,7 @@
  */
 abstract class Controller
 {
-    abstract public function __invoke(Request $request, Response $response): Response;
+    abstract public function __invoke(Request $request, Response $response, $args): Response;
```

Comment:
You can add this with a default empty array so that you don't need to edit all of the controllers - I was asked about something like this earlier so I must've misunderstood, sorry!

```suggestion
    abstract public function __invoke(Request $request, Response $response, $args = []): Response;
```

Reply:
done

Reply:
Left as is, as the controllers required changing anyway